snoop/defaultsettings.py

```diff
@@ -169,12 +169,8 @@
 """Django locale."""
 
 DETECT_LANGUAGE = True
-"""Enable language detection."""
-
-LANGUAGE_DETECTOR_NAME = 'polyglot'
-"""Configure which language detector library to use.
-"""
-
+EXTRACT_ENTITIES = True
+NLP_SERVICE_URL = os.environ.get('SNOOP_NLP_URL', 'http://127.0.0.1:5000/')
```

Comment:
docstrings for all 3 of these pls.

remember docstrings for assignments are after the assignment (see rest of this file)